### PR TITLE
Content descriptors added for supporting talkback accessibility

### PR DESCRIPTION
**Pull request by AlexMavDev**
* getCategoryName method added in EmojiCategory interface to allow getting the category name.
* getView method changed to set content descriptor for each EmojiImageView ('button') in EmojiArrayAdapter.
* inflateButton method changed to set the content descriptor in ImageButton used for the each emoji category tab.
* Categories classes changed to implement getCategoryName method from EmojiCategory.
* Strings files added containing category name strings.

### CODE REVIEW

**Comment by mario** [comment]
Hello @AlexMavDev. First of all, thank you for your great contribution.

Two things:
1) I'm not sure the "_name" suffix is needed in string resources
2) Would you be open to modifying https://github.com/vanniktech/Emoji/blob/85ce9b47fdf666edd4d3b76874cbbd06e3c6bdf4/generator/template/Category.java so that the getCategoryName is generated automatically once we do a re-pull of our data source, and don't have to do it manually every time?
3) That being said, strings.xml can be automatically generated too but we can leave that to a future iteration :)

**Comment by AlexMavDev** [comment]
Hi @mario 
Changes done. What Category class is used for?

**Comment by mario** [comment]
@AlexMavDev it's used to generate files/classes like NatureCategory :)

File: emoji/src/main/java/com/vanniktech/emoji/emoji/EmojiCategory.java [hunk 1]
```diff
@@ -22,4 +23,11 @@
    * @since 0.4.0
    */
   @DrawableRes int getIcon();
+
+  /**
+   * Returns category name.
+   *
+   * @since
```
Comment (vanniktech) [inline emoji/src/main/java/com/vanniktech/emoji/emoji/EmojiCategory.java]:
```suggestion
   * @since 0.7.0
```

Comment (AlexMavDev) [reply emoji/src/main/java/com/vanniktech/emoji/emoji/EmojiCategory.java]: Thanks a lot guys :)

**Comment by AlexMavDev** [comment]
> @AlexMavDev it's used to generate files/classes like NatureCategory :)

Ok. Thanks @mario

**Comment by scalajay** [comment]
Will this be merged anytime soon?

**Comment by vanniktech** [comment]
Thanks @rubengees :)

**Comment by scalajay** [comment]
Hi all, can we get a new release? Im really looking forward to using this accessibility patch. Thank you.

**Comment by vanniktech** [comment]
I'd like to have https://github.com/vanniktech/Emoji/pull/346 in the next release but it still needs some work

**Pull request merged**